src/govuk/components/button/_index.scss

```diff
@@ -1,5 +1,16 @@
+////
+/// @group components/button
+////
+
+/// Button component primary background colour
+///
+/// @type Colour
+/// @access public
+
+$govuk-button-background-colour: govuk-colour("green", $legacy: #00823b) !default;
```

Comment:
Would this line be better placed in `src/govuk/settings/_colours-applied.scss`? That's where other default variables definitions seem to be located. 

I'm conscious that this variable is specific to the button component, however, and we don't seem to have precedent for where to put them specifically. Happy to keep it here if we think that makes more sense.

Reply:
I'm thinking over time I can contribute the different high level component variables based on me branding the GOV.UK Design System so it'll make more sense.